.github/actions/build/action.yml

```diff
@@ -12,6 +12,8 @@ runs:
       dotnet-version: |
         6.x
         7.x
+        8.0.100-alpha.1.23061.8
```

Comment:
How often do we expect to keep this version number updated? Or is it appropriate to use latest pre-release?

Reply:
I don't expect this number to change until .NET 8 goes GA. Taking the latest SDK is problematic as it burns the version into the artifacts, and that then requires consumers to have at least the same version of runtime installed locally. See https://github.com/microsoft/slngen/issues/456.